src/Extension/ValueResolvers/CustomOptionsValueResolver.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Extension\ValueResolvers;
+
+use SprykerSdk\Sdk\Core\Appplication\ValueResolver\AbstractValueResolver;
+
+class CustomOptionsValueResolver extends AbstractValueResolver
+{
+    /**
+     * @return string
+     */
+    public function getId(): string
+    {
+        return 'CUSTOM_OPTIONS';
+    }
+
+    /**
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return 'Additional options for task';
+    }
+
+    /**
+     * @return string
+     */
+    public function getType(): string
+    {
+        return 'string';
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getAlias(): ?string
```

Comment:
It's not necessary to mention "null" here. Is it a convention to keep the parent's return type as it is?

Reply:
The idea to have in getValue somthing like this 
`{
if ($value === null) {
return null;
}

return sprintf('--%s=%s', $optionName, $value);
}`

Reply:
Fixed